.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Crash Support/Troubleshooting
+about: Get support for a game that won't load or is crashing
+title: "[Support] [TITLE OF YOUR ISSUE]"
+labels: support
+---
+The issues of the fabric GitHub repository is not the best place for you to get support for your issue. If you would like better and more immediate help with your issue please join the fabric discord server [here](https://discord.gg/v6v4pMv), and ask for help in the #player-support channel.
+
+If you are not getting the help you need on the discord server then try the following steps to resolve the issue yourself.
+
+1. Check that your fabric loader installation is correct by removing all of the mods from your mod folder and running the game. If the game successfully loads continue to the next step. If not, then read the instructions [here](https://fabricmc.net/wiki/install) on how to properly install the fabric loader.
+2. Remove all of the mods from your mod folder, and add them back one by one until you find which mod is the culprit. When you find which mod is responsible for the crashing, then look for a link to the sources for the mod and make an issue on the repository for that mod.
```

Comment:
I'd probably throw a space between each step so it doesn't look so wall of texty

Reply:
Should be resolved in https://github.com/FabricMC/fabric/pull/1051/commits/e4db6628a84f38b0e2acf8116790ea7088bdb2ed